six/modules/python/six.sicd/tests/test_create_sicd_xml.py

```diff
@@ -34,686 +35,777 @@
-for i in range(4):
-    for j in range(4):
-        pfa.slowTimeDeskew.slowTimeDeskewPhasePoly[(i, j)] = 44 * i + j
-
-cmplx.pfa = pfa
-
-### RMA ###
-
-rma = makeScopedCopyableRMA()
-rma.algoType = RMAlgoType('OMEGA_K')
-rma.rmat = makeScopedCopyableRMAT()
-rma.rmat.refTime = 12
-for i in range(3):
-    rma.rmat.refPos[i] = i
-    rma.rmat.refVel[i] = i + 1
-rma.rmat.distRefLinePoly = Poly1D(3)
-for i in range(4):
-    rma.rmat.distRefLinePoly[i] = i * 5
-rma.rmat.cosDCACOAPoly = Poly2D(3, 3)
-for i in range(4):
-    for j in range(4):
-        rma.rmat.cosDCACOAPoly[(i, j)] = i + j
-rma.rmat.kx1 = 1
-rma.rmat.kx2 = 2
-rma.rmat.ky1 = 3
-rma.rmat.ky2 = 4
-rma.rmat.dopConeAngleRef = 5
-
-# Just make this one anyway to make sure it works
-rmcr = makeScopedCopyableRMCR()
-for i in range(3):
-    rmcr.refPos[i] = i
-    rmcr.refVel[i] = i * 2
-rmcr.dopConeAngleRef = 9
-
-# Just make this one anyway to make sure it works
-inca = makeScopedCopyableINCA()
-inca.timeCAPoly = Poly1D(3)
-for i in range(3):
-    inca.timeCAPoly[i] = i
-inca.rangeCA = 5
-inca.freqZero = 10
-inca.dopplerRateScaleFactorPoly = Poly2D(3, 3)
-inca.dopplerCentroidPoly = Poly2D(3, 3)
-for i in range(4):
-    for j in range(4):
-        inca.dopplerRateScaleFactorPoly[(i, j)] = 9 * i
-        inca.dopplerCentroidPoly[(i, j)] = 10 * i
-inca.dopplerCentroidCOA = BooleanType('IS_TRUE')
-
-# Don't actually assign RMA to cmplx since we're using PFA
-
-### RgAzComp ###
-
-rgAzComp = makeScopedCopyableRgAzComp()
-rgAzComp.azSF = 123
-rgAzComp.kazPoly = Poly1D(3)
-for i in range(4):
-    rgAzComp.kazPoly[i] = i * 100
-
-# Don't actually assign RgAzComp to cmplx since we're using PFA
-
-### Now format it as XML and write it out to a file ###
-vs = VectorString()
-vs.push_back(os.environ['SIX_SCHEMA_PATH'])
-  
-xml_ctrl = ComplexXMLControl()
-origPathname = 'test_create_sicd.xml'
-fos = FileOutputStream(origPathname)
-out_doc = xml_ctrl.toXML(cmplx, vs)
-root = out_doc.getRootElement()
-root.prettyPrint(fos)
-fos.close()
-
-# If we made it to here, all the Python bindings must be present and what we
-# wrote out must have passed schema validation
-# We can't tell if for some reason some of the XML just didn't get written
-# out though
-
-### Now read it back in again ###
-
-fis = FileInputStream(origPathname)
-xmlparser = MinidomParser()
-xmlparser.preserveCharacterData(True)
-xmlparser.parse(fis)
-in_doc = xmlparser.getDocument()
-
-data = xml_ctrl.fromXML(in_doc, vs) 
-cmplxReadBackIn = asComplexData(data)
-
-# And then write it out one more time #
-newPathname = 'test_create_sicd_rt.xml'
-fos = FileOutputStream(newPathname)
-out_doc = xml_ctrl.toXML(cmplxReadBackIn, vs)
-root = out_doc.getRootElement()
-root.prettyPrint(fos)
-
-# These should match #
-if filecmp.cmp(origPathname, newPathname):
-    print 'Round trip succeeded!'
-else:
-    print 'Round trip failed'
-
-# If we made it to here, the read side appears to be working properly too
+def initCollectionInfo(cmplx):
```

Comment:
Did line endings change or why does every line show up as a diff?  Seems like the old file already had Unix line endings...